cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -2491,6 +2496,21 @@ impl MachInst for Inst {
         }
     }
 
+    fn is_included_in_clobbers(&self) -> bool {
+        // We exclude call instructions from the clobber-set when they are calls
+        // from caller to callee with the same ABI. Such calls cannot possibly
+        // force any new registers to be saved in the prologue, because anything
+        // that the callee clobbers, the caller is also allowed to clobber. This
+        // both saves work and enables us to more precisely follow the
+        // half-caller-save, half-callee-save SysV ABI for some vector
+        // registers.
+        match self {
+            &Inst::Call { ref info } => info.caller_callconv != info.callee_callconv,
+            &Inst::CallInd { ref info } => info.caller_callconv != info.callee_callconv,
```

Comment:
uber-nit: can you join the two cases, please? `&Inst::Call { ref info } | &Inst::CallInd { ref info }`

Reply:
Unfortunately they're different types (`CallInfo` vs. `CallIndInfo`) so we can't unify the patterns here :-/